src/Platform/Microsoft.Testing.Platform/Services/ITestSessionContext.cs

```diff
@@ -5,9 +5,19 @@
 
 namespace Microsoft.Testing.Platform.Services;
 
-internal interface ITestSessionContext
+/// <summary>
+/// Interface representing the context of a test session.
+/// It's used in ITestSessionLifetimeHandler to provide information about the session.
+/// </summary>
+public interface ITestSessionContext
 {
-    SessionUid SessionId { get; }
+    /// <summary>
+    /// Gets a unique identifier for the test session.
+    /// </summary>
+    SessionUid SessionId { get; } // TODO: Should this rename to SessionUid?
```

Comment:
I'd prefer to address this TODO in this PR and not leave it once we decide.

Reply:
Makes sense to me, it's clearer.

Cc @MarcoRossignoli